src/server/utils/capio_file.hpp

```diff
@@ -92,6 +93,21 @@ class Capio_file {
         }
     }
 
+    inline void set_complete() {
```

Comment:
Please remove this method and make `_complete` default to `true`

```c++
inline void set_complete(bool _complete = true) {
   ...
}
```

Reply:
This should not be done, as if a capio file is set to true by default, then a file that has just been created but not yet written will be available to other applications, completely ignoring the semantic specified on the file. The correct behavior is the one implemented.